frontend/src/components/Skill/types.ts

```diff
@@ -0,0 +1,11 @@
+import { SkillsList } from 'types/skill';
+
+export interface ISkillsGroupProps {
```

Comment:
as i see this props never overused so write them directly in components where you use them